docs/AQL/master02-overview.adoc

```diff
@@ -31,17 +31,16 @@ Below is an example of AQL statement. This statement returns all blood pressure
 
 ----
 SELECT                                                       -- Select clause
-    o/data[at0001]/.../items[at0004]/value AS systolic,      -- Identified path with named result
-    o/data[at0001]/.../items[at0005]/value AS diastolic,
+   o/data[at0001]/.../items[at0004]/value AS systolic,      -- Identified path with named result
```

Comment:
maybe re-align the comments on the right in this block.

Reply:
Good, comments are aligned.